src/routes/blog/posts/introducing-files-v2-4/+page.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Introducing v2.4
+description: Taking a look at the features and improvements in Files v2.4
+thumbnail: /blog-resources/introducing-v2-4-preview/HeroImage.jpg
+date: 2022-12-26
+author: files-community
+---
+
+We're pleased to announce that after a couple of weeks, v2.4 is available for all our users. Files has auto update built-in but if you're a new user or looking to manually update, you can download Files from our new [download page](/download/). Improved performance when navigating between directories, new features for managing archives and design changes are just some of the highlights for this release.
+
+## Supporting Files
+In an effort to keep maintaining Files, we recently started charging for the Store version. Purchasing Files through the Store is the easiest way to help support the project, but we want to make users aware that this is optional and that Files can still be installed for free by using the [sideload](/download/) version. We remain committed to open-source and will continue to publish our code on GitHub as usual.
+
+
+## Highlights
+
+### Design
+- Switched to using Mica Alt in the title bar for better contrast between the tabs
+<figure class="margin-bottom">
+    <img src="/blog-resources/filesv2-4/MicaAlt.png" alt="A screenshot Mica Alt in the title bar." />
```

Comment:
No need to specify "a screenshot". Screen readers already announce `<img>` tags as images. A handy trick I remember is to write as if I was describing the image to a friend: "An image of: …".